File: 3rd_sem_sxc/SEM==3/PERL/fibo.pl
# To find the fibonacci series
# Q - 2

	system("clear");
	printf("\n Enter any +ve number which will be the limit of the fibonacci number :");
	$n=<STDIN>;
	$a=0;
	$b=1;

	printf("%d ",$a);
	$c=0;
	while ($c < $n)
	{
		
		printf(" %d ",$c);
		$a=$b;
		$b=$c;
		$c=$a+$b;
		
	}

File: java_backup/my java/AYAN/fibo.java
import java.io.*;
class fibonacci
{
public static void main(String args[])throws IOException
{
BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
int a=0,b=1,s=0,n=0;
System.out.println("Enter any number:");
n=Integer.parseInt(br.readLine());
System.out.println("The fibonacci series is:");
System.out.println(a);
System.out.println(b);
while(s<=n)
{
s=a+b;
a=b;
b=s;
System.out.println(s);
}
}
}
